src/ng2-modal.ts

```diff
@@ -1,7 +1,7 @@
 import {Component, Input, Output, EventEmitter, ElementRef, ViewChild} from "@angular/core";
 
 @Component({
-    selector: "modal",
+    selector: "ng2-modal",
```

Comment:
I don't think it's any better. It's compliant for the upcoming webcomponents spec draft from w3c. In the new draft any custom elements must contain a dash. The reason they claim is so there will be less conflicts between spec elements and user custom elements.